### PR TITLE
Null Safety fix

### PR DESCRIPTION
**Pull request by oakstair**

### CODE REVIEW

**Comment by oakstair** [comment]
Shit! Seems like something has to be done with CIrcleCI setup. It built locally. 

Seems like the dart version has to be upped in CircelCI. Not my cup of tea :-)

**Comment by debkanchan** [comment]
+1 bump please look into this @ammaratef45

**Comment by ammaratef45** [comment]
@oakstair @DebkanchanSamadder 
I've updated the docker image but I'm facing a bug with CircleCI (https://github.com/circleci/circleci-docs/issues/5144)

I will wait for a day and if there's no solution I will run the workflow locally and merge if all is good

**Comment by debkanchan** [comment]
> @oakstair @DebkanchanSamadder 
> I've updated the docker image but I'm facing a bug with CircleCI (https://github.com/circleci/circleci-docs/issues/5144)
> 
> I will wait for a day and if there's no solution I will run the workflow locally and merge if all is good

Can you update the config to use cirrusci/flutter image and see if that works?

**Comment by ammaratef45** [comment]
> > @oakstair @DebkanchanSamadder
> > I've updated the docker image but I'm facing a bug with CircleCI ([circleci/circleci-docs#5144](https://github.com/circleci/circleci-docs/issues/5144))
> > I will wait for a day and if there's no solution I will run the workflow locally and merge if all is good
> 
> Can you update the config to use cirrusci/flutter plugin and see if that works?

@DebkanchanSamadder That worked, thanks a lot

@oakstair can you update your fork with the latest commit?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 0.1.4
+Null Safety fix
```
Comment (debkanchan) [inline CHANGELOG.md:2]: A better description would be `Migrate to Sound Null Safety`

**Comment by debkanchan** [comment]
Hey @ammaratef45 is it ok if I do a similar PR and then you can do a release? I need this to be null safe as soon as possible for my production app

**Comment by ammaratef45** [comment]
@DebkanchanSamadder Go ahead 👍

**Comment by ammaratef45** [comment]
@DebkanchanSamadder @oakstair Thank you both for looking into this

**Comment by ammaratef45** [comment]
@oakstair @DebkanchanSamadder 
`0.1.4` is now released 🚀

**Pull request merged**